.github/workflows/python_package.yml

```diff
@@ -89,6 +116,7 @@ jobs:
         run: |
           docker run \
             --rm \
+            --env CMAKE_BUILD_PARALLEL_LEVEL=${{ env.CMAKE_BUILD_PARALLEL_LEVEL }} \
```

Comment:
This is one that I missed in #6458, noticed it when replicating some of the configuration for the new CI job here